### ISSUE TITLE
poor contrast for input placeholder text

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/hourglass#245]
## Report Issue
**Describe the inaccessible feature**
Input placeholder text fails the colour contrast requirements, being 2.7:1.  This only seems to apply to placeholders, actual typed text into those fields is darker and meets the threshold.

**To Reproduce**
Steps to reproduce:
1. Go to [resources page]( https://olhbeta.janeway.systems/site/resources/)
2. see error in the placeholder 'Filter' contrast with its background

**Screenshots**
<img width="290" alt="image" src="https://github.com/BirkbeckCTP/hourglass/assets/5330770/d504f246-5a3d-43fa-9133-b6fdf88eb25a">

## Preferred Solution
colour of this placeholder text needs to change to meet the contrast requirement 4.5:1.

## Additional context
This issue is repeated on other pages which have inputs with placeholders, e.g. Journals page.

**Comment by StephDriver** [comment]
https://www.w3.org/WAI/tutorials/forms/instructions/#placeholder-text. for guidance.  as 'web browsers usually display the placeholder text in a color that does not meet the minimum contrast requirement of WCAG'.
